controllers/rhmi/rhmi_controller.go

```diff
@@ -307,9 +307,11 @@ func (r *RHMIReconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
 	installationSKU := &sku.SKU{}
 	if installation.Spec.Type == string(rhmiv1alpha1.InstallationTypeManagedApi) {
 		if err = r.processSKU(installation, request.Namespace, installationSKU); err != nil {
```

Comment:
Just an option here, I was wondering if we could move this to the [bootstrap reconciler ](https://github.com/integr8ly/integreatly-operator/blob/master/controllers/rhmi/bootstrapReconciler.go#L66)  

it would help to decrease the size of the rhmi_controller and as it seems the bootstrap phase is where the configuration to install the operator happens